### PR TITLE
Integrate subgraph repo to htm_escrow

### PR DESCRIPTION
**Pull request by Charlie85270**
@posix4e 

This PR fix : 

- https://github.com/humanprotocol/hmt-escrow/issues/333
- https://github.com/humanprotocol/humansubgraph/issues/16
- https://github.com/humanprotocol/humansubgraph/issues/7

### CODE REVIEW

**Comment by posix4e** [comment]
@ivhus @Dzeranov he already signed nda on another repo so no worries

**Comment by ivhus** [comment]
`.github` part looks fine to me, idk much about internals of other specific parts though. CI+CD approved

**Review by ivhus — APPROVED** [review]

**Review by ivhus — COMMENTED** [review]
binaries and temporary files like .DS_Store shouldn't be included in version control and also those are better to add to `.gitignore` @Charlie85270

**Comment by Charlie85270** [comment]
Updated

**Review by posix4e — CHANGES_REQUESTED** [review]
There’s no need to check in the Abi. It should be generated

**Comment by Charlie85270** [comment]
> There’s no need to check in the Abi. It should be generated

@posix4e What do you mean by check ? versioned ? 

If yes, indeed we can un-versioned ABI files because they are automatically generated before run/build the subgraph.

File: hmt_subgraph/abis/EscrowFactory.json [hunk 1]
```diff
@@ -0,0 +1,147 @@
+[
```
Comment (posix4e) [inline hmt_subgraph/abis/EscrowFactory.json]: Don’t check this in

File: hmt_subgraph/abis/Escrow.json [hunk 2]
```diff
@@ -0,0 +1,453 @@
+[
```
Comment (posix4e) [inline hmt_subgraph/abis/Escrow.json]: Don’t check this in

**Review by posix4e — CHANGES_REQUESTED** [review]

**Review by posix4e — APPROVED** [review]

**Comment by Charlie85270** [comment]
Updated, I removed the generated ABI files.

**Pull request merged**